<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPasswordDlg</class>
 <widget class="BaseDlg" name="ModifyPasswordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModifyPasswordDlg</string>
  </property>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>450</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>450</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QFrame" name="GridFrame">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>88</y>
     <width>572</width>
     <height>381</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>500</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="ModPageUserNameLab">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>571</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ModPageUserNameEdit">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>40</y>
      <width>570</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>570</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="ModPageOldkeyLab">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>86</y>
      <width>561</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>原密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ModPageOldkeyEdit">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>125</y>
      <width>570</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>570</width>
      <height>40</height>
     </size>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ModPageNewkeyLab">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>171</y>
      <width>571</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>新密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ModPageNewkeyEdit">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>210</y>
      <width>570</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>570</width>
      <height>40</height>
     </size>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ModPageNewkeySureLab">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>256</y>
      <width>571</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ModPageNewkeySureEdit">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>295</y>
      <width>570</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>570</width>
      <height>40</height>
     </size>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ErrTipLab">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>345</y>
      <width>341</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:14px;
color:#df173e;</string>
    </property>
    <property name="text">
     <string>错误提示</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>GridFrame</zorder>
  <zorder>CancelBtn</zorder>
  <zorder>OkBtn</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
